src/Promise/FulfilledPromise.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Http\Client\Utils\Promise;
+
+use Http\Client\Exception;
+use Http\Client\Promise;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * A promise already fulfilled
+ *
+ * @author Joel Wurtz <jwurtz@jolicode.com>
+ */
+class FulfilledPromise implements Promise
```

Comment:
do we need dedicated classes for this? would a "normal" Promise not happen to act the right way given the response? or how does a "normal" promise get the response?